src/Datadog.Trace/Conventions/IOutboundHttpConvention.cs

```diff
@@ -0,0 +1,12 @@
+using Datadog.Trace.Tagging;
+
+namespace Datadog.Trace.Conventions
+{
+    /// <summary>
+    /// Convention used to when defining operation names, span tags, statuses for outbound HTTP requests.
+    /// </summary>
+    internal interface IOutboundHttpConvention
+    {
+        Scope CreateScope(OutboundHttpArgs args, out HttpTags tags);
```

Comment:
Depending on how this grows to cover other operations the name may need to change to something more specific. eg.: `tracer.Convention.CreateOutboundHttpScope(...)`

Reply:
Or a facade will have such names.